docs/UserGuide.adoc

```diff
@@ -509,17 +565,19 @@ You are advised to key in a number smaller than 10000 due to size limitation.
 ==== Add a Sales Item to the Table
 This command adds a sales item into the table.
 
-* Command format:
+* Command:
 `add [c/CATEGORY] d/DESCRIPTION q/QUANTITY`
 
-The quantity of the item inputted must not be greater than the available stock in the inventory. Else, Leo will display
-a message prompting input of a smaller quantity or another item. +
+The quantity of the item must not be greater than the stock available in the Inventory Tab.
+Else, Leo will display a message prompting input of a smaller quantity or another item.
+
 The category field is optional. If you are unsure about the description of the desired item, you can refer to
-the Inventory tab or simply key in the category without any other fields. Leo will display all the items in the
+the Inventory tab or simply key in the category without any other fields. +
```

Comment:
I think you missed this line. "t" of "tab" should be in capital letters too right?

Reply:
Yes, thank you!